### PR TITLE
User Story 208240

### PR DESCRIPTION
**Pull request by TomArcherMsft**
[User Story 208240](https://dev.azure.com/msft-skilling/Content/_workitems/edit/208240)

### CODE REVIEW

File: quickstart/101-azure-api-management-create-with-api/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  value = {
+    id             = azurerm_api_management_api.api.id
+    is_current     = azurerm_api_management_api.api.is_current
+    is_online      = azurerm_api_management_api.api.is_online
+    version        = azurerm_api_management_api.api.version
+    version_set_id = azurerm_api_management_api.api.version_set_id
+  }
+}
+
+output "product_ids" {
```
Comment (lonegunmanb) [inline quickstart/101-azure-api-management-create-with-api/outputs.tf]: It seems like one id, could we rename it to `product_id`?

File: quickstart/101-azure-api-management-create-with-api/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  description = "The ID of the Product created"
+  value       = azurerm_api_management_product.product.id
+}
+
+output "product_api_ids" {
```
Comment (lonegunmanb) [inline quickstart/101-azure-api-management-create-with-api/outputs.tf]: Could we rename it to `product_api_id`?

File: quickstart/101-azure-api-management-create-with-api/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  description = "The ID of the Product/API association created"
+  value       = azurerm_api_management_product_api.product_api.id
+}
+
+output "product_group_ids" {
```
Comment (lonegunmanb) [inline quickstart/101-azure-api-management-create-with-api/outputs.tf]: Could we rename it to `product_group_id`

**Review by lonegunmanb — COMMENTED** [review]
Thanks @TomArcherMsft for opening this pr! A few questions on outputs.

File: quickstart/101-azure-api-management-create-with-api/outputs.tf (hunk 1 above)
Comment (TomArcherMsft) [reply quickstart/101-azure-api-management-create-with-api/outputs.tf]: Good catches. Fixed!

**Review by lonegunmanb — APPROVED** [review]
Thanks @TomArcherMsft , LGTM!

**Pull request merged**